cuda_bindings/cuda/bindings/_lib/windll.pxd

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: Copyright (c) 2021-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+from libc.stddef cimport wchar_t
+from libc.stdint cimport uintptr_t
+from cpython cimport PyUnicode_AsWideCharString
+
+cdef extern from "windows.h":
+    ctypedef void* HMODULE
+    ctypedef void* HANDLE
+    ctypedef void* FARPROC
+    ctypedef unsigned long DWORD
+    ctypedef const wchar_t *LPCWSTR
+
+    cdef DWORD LOAD_LIBRARY_SEARCH_SYSTEM32 = 0x00000800
+
+    HMODULE _LoadLibraryExW "LoadLibraryExW"(
+        LPCWSTR lpLibFileName,
+        HANDLE hFile,
+        DWORD dwFlags
+    ) nogil
+
+    FARPROC _GetProcAddress "GetProcAddress"(HMODULE hModule, const char* lpProcName) nogil
```

Comment:
As mentioned in the other MR, I think this should be
```suggestion
    FARPROC _GetProcAddress "GetProcAddress"(HMODULE hModule, LPCWSTR lpProcName) nogil
```

Reply:
Ah... I misread `LPCSTR` as `LPCWSTR`, so this was in fact correct. Thanks for not following my wrong comment 😅